HISTORY.rst

```diff
@@ -2,6 +2,31 @@
 History
 =======
 
+2.4.0 (2019-03-18)
+------------------
+
+Critical changes:
+
+* There have been some significant changes to the task for publishing plans to MetaDeploy:
```

Comment:
Should these be covered in a doc somewhere instead of just here in release notes?